ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.js

```diff
@@ -187,32 +187,32 @@
     },
     {
       id: 'UWU Garuda Woken',
-      regex: / 1A:\y{ObjectId}:Garuda gains the effect of Woken from/,
-      regexCn: / 1A:\y{ObjectId}:迦楼罗 gains the effect of 觉醒 from/,
-      regexDe: / 1A:\y{ObjectId}:Garuda gains the effect of Ätherüberladung from/,
-      regexFr: / 1A:\y{ObjectId}:Garuda gains the effect of Éveil éthéré from/,
-      regexJa: / 1A:\y{ObjectId}:ガルーダ gains the effect of 覚醒 from/,
-      regexKo: / 1A:\y{ObjectId}:가루다 gains the effect of 각성 from/,
+      regex: Regexes.gainsEffect({ effect: 'Woken', capture: false }),
```

Comment:
These should have `target: 'Garuda',` in them (and also the Ifrit and Titan ones below, similarly).

Reply:
ok added garuda, titan and ifrit